### ISSUE TITLE
Improvements in code and suggestions

### ISSUE THREAD

**Issue by yedhink** [neetozone/neeto-editor#150]
https://www.loom.com/share/4928c733a3014eb19e614c2d7eab0cf4

@AbhayVAshokan _a

Please take this up and assign to your tech lead once done.

cc: @labeebklatif

**Comment by karthiknmenon** [comment]
@yedhink 
I am putting this issue on hold right now. The reason being that neetoEditor TipTap is still in the growing stages. 
Our initial motive was to get the basics working. Then we added features that were necessary and now we are slowly porting it to different neeto products. 
And the process is a WIP. As products are porting from Trix editor to neetoEditor TipTap, we are coming across a lot of different issues. It could be style or logic based. And we need to prioritise on those issues for now. 

We can pick this up once the major work for supporting all neeto products is done and dusted.

cc: @labeebklatif @amaldinesh7

**Comment by labeebklatif** [comment]
taking this up

@labeebklatif _a

cc: @karthiknmenon @AbhayVAshokan
